LoopUI/Extensions/ChartPoint.swift

```diff
@@ -38,22 +39,44 @@ extension ChartPoint {
                 endDate = ChartAxisValueDate(date: targetRanges[index + 1].startDate, formatter: dateFormatter)
             }
 
-            let value = range.value.doubleRangeWithMinimumIncrement(in: unit)
-            let minValue = ChartAxisValueDouble(value.minValue)
-            let maxValue = ChartAxisValueDouble(value.maxValue)
-
-            maxPoints += [
-                ChartPoint(x: startDate, y: maxValue),
-                ChartPoint(x: endDate, y: maxValue)
-            ]
-
-            minPoints += [
-                ChartPoint(x: startDate, y: minValue),
-                ChartPoint(x: endDate, y: minValue)
-            ]
+            if let override = override, startDate.date < endDate.date {
```

Comment:
we should avoid using key terms when naming variables.

Reply:
Duh, and I gave that very same advice in a PR review just recently 🤦